### DISCUSSION TITLE
Diagnostic Tool is not supported?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaUI/AvaloniaVS#439] [Q&A]
I see the following message when debugging my Avalonia Android/iOS app, clicking on Debug > Windows > Show Diagnostic Tools.

![image](https://github.com/AvaloniaUI/AvaloniaVS/assets/1595223/2eb1ab4d-8142-473b-a5eb-cfda367ecd13)

Can I enable it somehow, or is it a limitation of AvaloniaVS?

**Comment by maxkatz6** [comment]
I don't think AvaloniaVS has anything to do with it. Does it work with plain android/ios app without Avalonia or MAUI?
